### PR TITLE
Support load_fred API

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

Support load_fred

# Checklist
Make sure you have performed the following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/system.R [hunk 1]
```diff
@@ -2953,3 +2953,21 @@ filter_cascade <- function(.data, ...){
   }
   df
 }
+
+#
+#'@export
+load_fred <- function(series_id, date_start = "", date_end = NULL, password) {
```
Comment (hideaki) [inline R/system.R:2959]: Why is date_start default is empty string while the date_end default is NULL?

File: R/system.R [hunk 2]
```diff
@@ -2953,3 +2953,21 @@ filter_cascade <- function(.data, ...){
   }
   df
 }
+
+#
+#'@export
```
Comment (hideaki) [inline R/system.R:2958]: Add some comment to describe this function.

File: R/system.R (hunk 1 above)
Comment (hidekoji) [reply R/system.R]: The default for date_end should be an empty string so I updated the default as "".

File: R/system.R (hunk 2 above)
Comment (hidekoji) [reply R/system.R:2961]: Added comment.

**Pull request merged**